rn/views/ProjectsView.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import { View } from "react-native";
+import { useAuth } from "../providers/AuthProvider";
+import { ListItem } from "react-native-elements";
+
+export function ProjectsView({ navigation, route }) {
+  const { projectData } = useAuth();
+
+  // the onClickProject navigates to the Task List with the project name
+  // and project partition value
+  const onClickProject = async (project) => {
+    try {
+      navigation.navigate("Task List", {
+        name: project.name,
+        projectPartition: project.partition,
+      });
+    } catch (err) {
+      throw `error opening user realm ${err}`;
```

Comment:
Generally, you don't want to throw strings but rather `new Error(...)`.

But if all this catch block is doing is rethrowing, you should remove the try/catch altogether.

Reply:
That makes sense, and you're right this try catch block should be removed. It's the remnants of when the realm was being opened here before. Fixed now :)